opm/simulators/flow/BlackoilModelParameters.cpp

```diff
@@ -0,0 +1,226 @@
+    tolerance_well_control_ = Parameters::Get<Parameters::ToleranceWellControl<Scalar>>();
+    max_welleq_iter_ = Parameters::Get<Parameters::MaxWelleqIter>();
+    use_multisegment_well_ = Parameters::Get<Parameters::UseMultisegmentWell>();
+    tolerance_pressure_ms_wells_ = Parameters::Get<Parameters::TolerancePressureMsWells<Scalar>>();
+    relaxed_tolerance_flow_well_ = Parameters::Get<Parameters::RelaxedWellFlowTol<Scalar>>();
+    relaxed_tolerance_pressure_ms_well_ = Parameters::Get<Parameters::RelaxedPressureTolMsw<Scalar>>();
+    max_pressure_change_ms_wells_ = Parameters::Get<Parameters::MaxPressureChangeMsWells<Scalar>>();
+    max_inner_iter_ms_wells_ = Parameters::Get<Parameters::MaxInnerIterMsWells>();
+    strict_inner_iter_wells_ = Parameters::Get<Parameters::StrictInnerIterWells>();
+    strict_outer_iter_wells_ = Parameters::Get<Parameters::StrictOuterIterWells>();
+    regularization_factor_wells_ = Parameters::Get<Parameters::RegularizationFactorWells<Scalar>>();
+    max_niter_inner_well_iter_ = Parameters::Get<Parameters::MaxNewtonIterationsWithInnerWellIterations>();
+    shut_unsolvable_wells_ = Parameters::Get<Parameters::ShutUnsolvableWells>();
+    max_inner_iter_wells_ = Parameters::Get<Parameters::MaxInnerIterWells>();
+    maxSinglePrecisionTimeStep_ = Parameters::Get<Parameters::MaxSinglePrecisionDays<Scalar>>() * 24 * 60 * 60;
+    min_strict_cnv_iter_ = Parameters::Get<Parameters::MinStrictCnvIter>();
+    min_strict_mb_iter_ = Parameters::Get<Parameters::MinStrictMbIter>();
+    solve_welleq_initially_ = Parameters::Get<Parameters::SolveWelleqInitially>();
+    update_equations_scaling_ = Parameters::Get<Parameters::UpdateEquationsScaling>();
+    use_update_stabilization_ = Parameters::Get<Parameters::UseUpdateStabilization>();
+    matrix_add_well_contributions_ = Parameters::Get<Parameters::MatrixAddWellContributions>();
+    check_well_operability_ = Parameters::Get<Parameters::EnableWellOperabilityCheck>();
+    check_well_operability_iter_ = Parameters::Get<Parameters::EnableWellOperabilityCheckIter>();
+    max_number_of_well_switches_ = Parameters::Get<Parameters::MaximumNumberOfWellSwitches>();
+    use_average_density_ms_wells_ = Parameters::Get<Parameters::UseAverageDensityMsWells>();
+    local_well_solver_control_switching_ = Parameters::Get<Parameters::LocalWellSolveControlSwitching>();
+    use_implicit_ipr_ = Parameters::Get<Parameters::UseImplicitIpr>();
+    nonlinear_solver_ = Parameters::Get<Parameters::NonlinearSolver>();
+    const auto approach = Parameters::Get<Parameters::LocalSolveApproach>();
+    if (approach == "jacobi") {
+        local_solve_approach_ = DomainSolveApproach::Jacobi;
+    } else if (approach == "gauss-seidel") {
+        local_solve_approach_ = DomainSolveApproach::GaussSeidel;
+    } else {
+        throw std::runtime_error("Invalid domain solver approach '" + approach + "' specified.");
```

Comment:
I'm sure we get it through some transitive include statement, but we should explicitly
```C++
#include <stdexcept>
```
to bring a declaration of the `std::runtime_error` type into scope.

Reply:
sure, just an oversight..